Persephone/MPDClient/MPDClient.swift

```diff
@@ -0,0 +1,212 @@
+//
+//  MPDClient.swift
+//  Persephone
+//
+//  Created by Daniel Barber on 2019/1/25.
+//  Copyright © 2019 Dan Barber. All rights reserved.
+//
+
+import Foundation
+import mpdclient
+
+class MPDClient {
+  var delegate: MPDClientDelegate?
+  static let stateChanged = Notification.Name("MPDClientStateChanged")
+  static let queueChanged = Notification.Name("MPDClientQueueChanged")
+  static let queuePosChanged = Notification.Name("MPDClientQueuePosChanged")
```

Comment:
Similar comment: could all these notifications based on strings either become an extension on Notification or turned into static properties of a common class?

So something like `Notification.Persephone.stateChanged` or `MPDNotifications.stateChanged`?

Reply:
Yes, this is the wrong place for these really.